swiftnav_console/logging_bar.py

```diff
@@ -97,6 +99,7 @@ def __init__(self):
 
     @classmethod
     def post_data_update(cls, update_data: Dict[str, Any]) -> None:
+        update_data[Keys.SBP_LOGGING_FORMAT_INDEX] = update_data[Keys.SBP_LOGGING_LABELS].index(update_data[Keys.SBP_LOGGING_FORMAT])
```

Comment:
it might be better to have this posted in capnproto but we construct the string model here so will just leave it here...